### PR TITLE
Add border and improve line breaks in classBox

### LINKED ISSUE #66: classBox missing borders
The `classBox` in data-types has no borders on the right and at the bottom. 
Is this intentional, or missing in CSS?

### PR DESCRIPTION
**Pull request by rettinghaus**
closes https://github.com/music-encoding/guidelines/issues/66

### CODE REVIEW

File: source/web/css/mei-screen.css [hunk 1]
```diff
@@ -192,6 +191,7 @@
 }
 
 .specPage .classBox .classHeading .classDesc {
+	display: inline-block;
```
Comment (bwbohl) [inline source/web/css/mei-screen.css]: I can see what this changes but I'd rather add o.5rem padding to `.specPage .classBox .classHeading` then `.classDesc` would always stay inline with `.classLabel` but no run into the right border when it is longish

**Comment by bwbohl** [comment]
moreover what about sth lik

```css
.classContent .def:nth-of-type(2n) {
    background-color: #f5f5f5;
}
```

File: source/web/css/mei-screen.css (hunk 1 above)
Comment (rettinghaus) [reply source/web/css/mei-screen.css]: This is just to keep `classDesc` better separated from `classLabel`.
It changes breaks from this:

image

to this:

image

**Comment by rettinghaus** [comment]
Thanks @bwbohl, but I want to keep this PR small. There are still many things to improve.

File: source/web/css/mei-screen.css (hunk 1 above)
Comment (bwbohl) [reply source/web/css/mei-screen.css]: This is true only for those `classDesc` that are too long to fit into the line.

Comment (rettinghaus) [reply source/web/css/mei-screen.css]: Ok, seems we cannot agree on this, so I removed it again.

**Review by bwbohl — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
@rettinghaus May I ask you for a short "before/after" screenshot, so it's easier to compare the result of the changes?

**Comment by rettinghaus** [comment]
before 

image

after 

image

**Review by kepper — APPROVED** [review]
I left that open to save space, but I'm fine with this change :-) Thanks for caring about this, @rettinghaus 👍

**Pull request merged**